src/components/Dashboard/TalkDashboard.vue

```diff
@@ -116,7 +115,8 @@ const filteredConversations = computed(() => store.getters.conversationsList.fil
 async function startMeeting() {
 	try {
 		const conversation = await store.dispatch('createGroupConversation', {
-			roomName: conversationName.value ?? t('spreed', 'Meeting'), // TRANSLATORS: Fallback for instant meeting name; it uses user language statically
+			// // TRANSLATORS: Section header for meeting-related settings; also a static name fallback for instant meeting conversation
```

Comment:
A typo or on purpose?
```suggestion
			// TRANSLATORS: Section header for meeting-related settings; also a static name fallback for instant meeting conversation
```

Reply:
a typo